src/main/java/htsjdk/samtools/cram/build/CramSpanContainerIterator.java

```diff
@@ -99,10 +99,7 @@ public Container next() {
                     throw new RuntimeException("No more containers in this boundary.");
                 }
                 
-                final long offset = seekableStream.position();
-                final Container c = ContainerIO.readContainer(cramHeader.getVersion(), seekableStream);
-                c.offset = offset;
-                return c;
+                return ContainerIO.readContainer(cramHeader.getVersion(), seekableStream);
```

Comment:
`readContainer()` takes care of setting the offset by querying the `SeekableStream`